shipyard_blueprints/snowflake/shipyard_snowflake/snowflake.py

```diff
@@ -118,13 +119,18 @@ def upload(
             table_name (str): The name of the Snowflake Table to, if it doesn't exist, it will be created
             insert_method (str): The method to use when inserting the data into the table. Options are replace or append Defaults to 'replace'
         """
-        if insert_method not in ["replace", "append"]:
+        if insert_method not in ["replace", "append", "add"]:
```

Comment:
not worth the change but worth noting ideally we should use a set for these types of checks. Slightly faster.
{"replace", "append", "add"}: